gradle/android-library-common.gradle

```diff
@@ -15,9 +15,9 @@ android {
   }
   kotlinOptions {
     jvmTarget = "1.8"
-  }
-
-  kotlinOptions {
-    freeCompilerArgs = ["-Xallow-result-return-type"]
+    freeCompilerArgs += [
+        "-Xopt-in=kotlinx.coroutines.ExperimentalCoroutinesApi",
+        "-Xallow-result-return-type",
```

Comment:
So we can use kotlin's `Result` as output and coroutines without warnings :)

Reply:
Man oh man you just got alllll new new toys :P